src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Services/DefaultRazorSemanticTokensInfoService.cs

```diff
@@ -19,8 +20,8 @@ internal class DefaultRazorSemanticTokensInfoService : RazorSemanticTokensInfoSe
         // We need to keep track of the last couple of requests for use in previousResultId,
         // but if we let the grow unbounded it could quickly allocate a lot of memory.
         // Solution: an in-memory cache
-        private static readonly MemoryCache<string, IReadOnlyList<uint>> _semanticTokensCache =
-            new MemoryCache<string, IReadOnlyList<uint>>();
+        private static readonly MemoryCache<string, IReadOnlyList<int>> _semanticTokensCache =
```

Comment:
lol you can have negative semantic tokens now?

Reply:
I can change it to `uint` before I ship `0.18` 🤣